src/screens/TurboVoteScreen.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, WebView, Text, View } from 'react-native';
+import { Button } from 'react-native-elements';
+import PropTypes from 'prop-types';
+import colors from '../styles/colors';
+
+class TurboVoteScreen extends React.Component {
+    static propTypes = propTypes;
+    goToSuccessScreen = () => this.props.navigation.navigate('VoterRegSuc');
+
+    render() {
+        return (
+            <View style={{ flex: 1 }}>
+                <WebView
+                    source={{
+                        uri: "https://inline.turbovote.org/?r=civic"
+                    }}
+                    style={{ flex: 1 }}
+                />
+                <Button {...this.sharedButtonProps}
+                    buttonStyle={{
+                        marginTop: 40,
+                    }}
+                    backgroundColor={colors.red}
+                    title="Done with Registration" onPress={this.goToSuccessScreen} />
+            </View>
```

Comment:
In thinking about users who may already be registered to vote, or those who don't feel like doing it immediately (since our onboarding already has a few steps) it would be great to have like a "no thanks" button that took users straight to the elections screen. An option for "I'm already registered" might be nice as well just so we have that information